### DISCUSSION TITLE
How much does the plugin load the server and waste RAM?

### DISCUSSION THREAD

**Discussion by CABi-Code** [plan-player-analytics/Plan#2990] [Q&A]
I recently found out that plugins that constantly write data to the database spend a lot of RAM and load weak hosting.

How does your plugin work? How much does it load the system?

**Comment by AuroraLS3** [comment]
It depends on how many players you get, but I haven't heard any complaints.

**Comment by Hutch79** [comment]
> I recently found out that plugins that constantly write data to the database spend a lot of RAM and load weak hosting.

I would say this greatly depends on the hosters setup.
If the Database is hosted on the same server as the Minecraft Server, and if the Hoster has to many Minecrafts servers on it, it can happen that the Disk just can't keep up with writing the data or the CPU gets overloaded.

Generally, PLAN doesn't use a lot of resources, at least on my network the memory and CPU usage are minimal.
